src/main/java/org/heigit/ohsome/ohsomeapi/output/metadataresponse/MetadataResponse.java

```diff
@@ -19,10 +19,12 @@
 
   @ApiModelProperty(notes = "License and copyright info", required = true, position = 0)
   private Attribution attribution;
-  @ApiModelProperty(notes = "Version of this api", required = true, position = 0)
+  @ApiModelProperty(notes = "Version of this api", required = true, position = 1)
   private String apiVersion;
+  @ApiModelProperty(notes = "Maximal response timeout in seconds", required = true, position = 2)
+  private double timeoutInSeconds;
```

Comment:
would it make sense to call this just `timeout` to be consistent with the naming of the query parameter: https://github.com/GIScience/ohsome-api/pull/73/files#diff-e18e0ca2c1587af2fb3550206da0889f4297c04bf4e3a1d1ee09ca46041a3c3dR25

Reply:
We don't have a description there in the /metadata response like we have it in other responses to describe in which unit the represented value is given, so that's why I guess it makes sense to include it here in the name.

We are also not always consistent with the unit, as in the startup parameter we use miliseconds and in the query parameter that you can set via the request we use seconds.

Reply:
well, there is https://docs.ohsome.org/ohsome-api/v1/endpoints.html#metadata-endpoint for documentation.

for "the other responses" you mean the `showMetadata=true` output… that's true. but still one has to look at the documentation in order to know what unit the `timeout` parameter's values have to be supplied in.

A point could also be made that the `timestamp` fields in the results are not "described" (via `showMetadata`), nor is the field called `timestampAsIso8601TimeString`. :thinking:

Reply:
Ah true, https://docs.ohsome.org/ohsome-api/v1/endpoints.html#metadata-endpoint this would actually also have to be adapted including the new parameter @bonaparten.
And yes, we can also add a output parameter description part then to https://docs.ohsome.org/ohsome-api/v1/response-parameters.html for the /metadata response where we can then describe the unit. If we have that, it's definitely fine to call it just "timeout".

For the time-related parameters we have https://docs.ohsome.org/ohsome-api/v1/time.html. So yeah we should just collect the relevant info in the docs, then it should be fine.